cc/algorithms/numerical-mechanisms_test.cc

```diff
@@ -268,6 +268,79 @@ TEST(NumericalMechanismsTest, GaussianMechanismAddsNoise) {
   EXPECT_FALSE(std::isnan(mechanism.AddNoise(1.1, -2.0)));
 }
 
+TEST(NumericalMechanismsTest, GaussNoiseConfidenceInterval) {
```

Comment:
Good idea! Tried to fix this :)

Reply:
There's an even better way to handle this sort of case: parameterized tests (see https://github.com/cpp-testing/GUnit/blob/master/docs/GTest.md, the last section). I'd suggest defining a struct to hold all the test parameters and using that as a parameter. It'll make things a bit neater than having a bunch of separate lists.

Reply:
Thanks for this comment! :) Changed the way I handled this case to use parameterized tests.

Reply:
Ah, this isn't quite what I was thinking. Rather than having a struct of lists, instead have each struct contain the parameters for one test, and then instantiate the test suite with a list of those structs. Then you won't need the for loop in the test.

Reply:
Gotcha, thanks! Will look into fixing this.

Reply:
I think I fixed it. Thanks again for bringing this up!